src/styles/blocks/_page.scss

```diff
@@ -0,0 +1,5 @@
+.page {
```

Comment:
no need to write a file name starting with the underscore `_` if it's a block. Only files in utils should be named in this way :slightly_smiling_face:

Reply:
I did this because if someone wants to convert scss to css, files with _ will not be read